oldabe/money_in.py

```diff
@@ -202,13 +212,18 @@ def renormalize(attributions, incoming_attribution):
     "renormalized."  This effectively dilutes the attributions by the magnitude
     of the incoming attribution.
     """
-    target_proportion = Decimal("1") - incoming_attribution.share
-    for email in attributions:
+    nondilutable_shares = [a.share for a in attributions.values() if not a.dilutable]
+    nondilutable_proportion = sum(nondilutable_shares)
+    target_proportion = (Decimal("1") - nondilutable_proportion - incoming_attribution.share) / (Decimal('1') - nondilutable_proportion)
```

Comment:
Maybe we should get the dilutable proportion here too instead of doing `Decimal("1") - nondilutable_proportion`